pages/events/fall-fest.vue

```diff
@@ -34,6 +34,7 @@
 
       <section class="fall-fest-page__section">
         <UiDataTableSection
+          class="fall-fest-page__data-table"
           :section-title="agenda.title"
           :data-table-columns="dataTable.headers"
```

Comment:
Here, we can "pass in" your description, into the `UiDataTableSection` component by using the `:section-description` prop... 

```suggestion
          class="fall-fest-page__data-table"
          :section-title="agenda.title"
          :section-description="agenda.description"
          :data-table-columns="dataTable.headers"
```

Reply:
resolved in https://github.com/Qiskit/qiskit.org/pull/3550/commits/9e7d698cbf2a0f898bd132f2575590cc22107a32